Update cannot-expand-nested-contact-groups-unknown-shown.md (#1726)
MicrosoftDocs/OfficeDocs-Support: Outlook/Client/contacts/cannot-expand-nested-contact-groups-unknown-shown.md

:::image type="content" source="media/cannot-expand-nested-contact-groups-unknown-shown/unknown-shown-for-nested-contact-group.png" alt-text="Nested Contact Group":::

## Resolution 
  
### Method 1

To resolve this issue, right-click the Contact folder in the Public Folder, and then select **Properties**. On the Outlook **Address Book** tab, select the **Show this folder as an email address book** option.
  
### Method 2
  

> 1) Open the group, Select **Update now**. If there are members that cannot be found, outlook will prompt you to choose between the three below options

Reviewer: ```suggestion
1) Open the group and select **Update now**. If there are members that cannot be found, Outlook will prompt you to choose between the following three options:
```

  
  - Remove the lost members.

>   - Try to repair  the lost members and remove only those that cannot be resolved.

Reviewer: ```suggestion
  - Try to repair the lost members and remove only those that cannot be resolved.
```

  - Cancel this operation and do not change any members.
 

> 2) Select, **Try to repair  the lost members and remove only those that cannot be resolved** and click **Ok**.

Reviewer: ```suggestion
2) Choose **Try to repair the lost members and remove only those that cannot be resolved** and select **Ok**.
```
